content/influxdb3/cloud-dedicated/admin/clusters/get.md

```diff
@@ -28,20 +28,8 @@ Use the Admin UI or the [`influxctl cluster get` CLI command](/influxdb3/cloud-d
 [influxctl](#)
 {{% /tabs %}}
 {{% tab-content %}}
-## Access the Cloud Dedicated Admin UI
-
-1. To access the {{< product-name >}} Admin UI, visit the following URL in your browser:
-
-   <pre>
-   <a href="https://console.influxdata.com">https://console.influxdata.com</a>
-   </pre>
-
-2. Use the credentials provided by InfluxData to log into the Admin UI.
-   If you don't have login credentials, [contact InfluxData support](https://support.influxdata.com).
-
-   After you log in, the Account Management portal displays [account information](/influxdb3/cloud-dedicated/admin/account/)
-   and lists all clusters associated with your account.
-3. **Search** for the cluster or use the sort button and column headers to sort the cluster list and find the cluster.
```

Comment:
We could leave all of this in, but I found this a little repetitive; for purposes of opening the application I think providing the URL and request to contact support if credentials are required is sufficient.